subsys/net/ip/Kconfig

```diff
@@ -683,6 +683,18 @@ config NET_CONTEXT_DSCP_ECN
 	  Notification values on net_context. Those values are then used in
 	  IPv4/IPv6 header when sending packets over net_context.
 
+config NET_CONTEXT_REUSEADDR
```

Comment:
Why are these even optional? Shouldn't it just always be supported in order to support this aspect of the BSD Socket API?

No Kconfig required, IMHO.